### ISSUE TITLE
Updated repo

### ISSUE THREAD

**Issue by andrewmclagan** [SamVerschueren/aws-lambda-stop-server#2]
Hello, no disrespect intended. This repo is now dated with the advent of node 4.x.x on lambda. With allot of inspiration from your work I added the following features:
- node ^4.x.x
- cross-region
- removed dependancies
- ES6 codebase
- modular, testable, functional and importantly readable codebase 

https://github.com/andrewmclagan/aws-lambda-ec2-start-stop

thanks for your hard work!!

**Comment by SamVerschueren** [comment]
Seems clean. You could've done a PR for that ;).

**Comment by andrewmclagan** [comment]
Was going to ... I could...?

**Comment by SamVerschueren** [comment]
Yeah sure, would be happy to accept it. It was written fairly quick because I needed it and open-sourced it for others to use it. But things can always be improved :).

**Comment by andrewmclagan** [comment]
Cool, will do that later in the week

**Comment by SamVerschueren** [comment]
Nice, looking forward to it :)!
